src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftBuildVerifier.java

```diff
@@ -49,111 +52,133 @@ public String getCheckForTriggeredDeployments() {
-         * @return
-         *      Indicates the outcome of the validation. This is sent to the browser.
-         *      <p>
-         *      Note that returning {@link FormValidation#error(String)} does not
-         *      prevent the form from being saved. It just means that a message
-         *      will be displayed to the user. 
-         */
-        public FormValidation doCheckApiURL(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckApiURL(value);
-        }
-
-        public FormValidation doCheckBldCfg(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckBldCfg(value);
-        }
-
-        public FormValidation doCheckNamespace(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckNamespace(value);
-        }
-        
-        public FormValidation doCheckCheckForTriggeredDeployments(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckCheckForTriggeredDeployments(value);
-        }
-        
-        public FormValidation doCheckAuthToken(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckToken(value);
-        }
-
-        public boolean isApplicable(Class<? extends AbstractProject> aClass) {
-            // Indicates that this builder can be used with all kinds of project types 
-            return true;
-        }
-
-        /**
-         * This human readable name is used in the configuration screen.
-         */
-        public String getDisplayName() {
-            return DISPLAY_NAME;
-        }
-        
-        public long getWait() {
-        	return wait;
-        }
-
-        @Override
-        public boolean configure(StaplerRequest req, JSONObject formData) throws FormException {
-            // To persist global configuration information,
-            // pull info from formData, set appropriate instance field (which should have a getter), and call save().
-        	wait = formData.getLong("wait");
-        	GlobalConfig.setBuildVerifyWait(wait);
-            save();
-            return super.configure(req,formData);
-        }
-
-    }
+	// If your plugin doesn't really define any property on Descriptor,
+	// you don't have to do this.
+	@Override
+	public DescriptorImpl getDescriptor() {
+		return (DescriptorImpl)super.getDescriptor();
+	}
+
+	/**
+	 * Descriptor for {@link OpenShiftBuildVerifier}. Used as a singleton.
+	 * The class is marked as public so that it can be accessed from views.
+	 *
+	 */
+	@Extension // This indicates to Jenkins that this is an implementation of an extension point.
+	public static final class DescriptorImpl extends BuildStepDescriptor<Builder>  implements TimedStepDescriptor {
```

Comment:
we've bee using the naming convention of having all interfaces start with `IOpenShift`.  Convert `TimedStepDescriptor` to `IOpenShiftTimedStepDescriptor`